src/components/calcite-action/calcite-action.tsx

```diff
@@ -123,15 +118,13 @@ export class CalciteAction {
   }
 
   render() {
-    const { compact, disabled, loading, el, textEnabled, textDisplay, label, text } = this;
+    const { compact, disabled, loading, el, textEnabled, label, text } = this;
 
-    const calculatedTextDisplay = textEnabled ? "visible" : textDisplay;
     const labelFallback = label || text;
     const rtl = getElementDir(el) === "rtl";
 
     const buttonClasses = {
-      [CSS.buttonTextVisible]: calculatedTextDisplay === "visible",
-      [CSS.buttonTextInteractive]: calculatedTextDisplay === "interactive",
```

Comment:
I'm thinking that textDisplay is more the appropriate property here since all actions should have text associated with them, yeah? And the property would be related to if and how the text is displayed.

Reply:
@asangma I was thinking we would go back to just textEnabled since we aren't using the animation in any use case. What do you think? I can put the animation back.

Reply:
Aren't we using the animation when the actionbar expands?

Reply:
yes but not on hover/focus. Thats what Im getting rid of.